Backend/app/dashboard/provider-dashboard/src/API/Fleet/Registration.hs

```diff
@@ -47,7 +47,13 @@ type API =
 type FleetOwnerRegisterAPI =
   "register"
     :> ReqBody '[JSON] DP.FleetOwnerRegisterReq
-    :> Post '[JSON] APISuccess
+    :> Post '[JSON] FleetOwnerRegisterResp
+
+data FleetOwnerRegisterResp = FleetOwnerRegisterResp
+  { status :: Text,
```

Comment:
maybe you meant `result :: Text` ?

because earlier it was `result`, not `status` :

```
instance A.ToJSON APISuccess where
  toJSON Success = A.object ["result" A..= ("Success" :: Text)]
```